cpp/ql/lib/semmle/code/cpp/security/InvalidPointerDereference/InvalidPointerToDereference.qll

```diff
@@ -99,59 +105,79 @@ private module InvalidPointerToDerefBarrier {
 
   private module BarrierFlow = DataFlow::Global<BarrierConfig>;
 
-  private int getInvalidPointerToDerefSourceDelta(DataFlow::Node node) {
-    exists(DataFlow::Node source |
-      BarrierFlow::flow(source, node) and
-      invalidPointerToDerefSource(_, _, source, result)
-    )
-  }
-
+  /**
+   * Holds if `g` ensures that `small < big + k` if `g` evaluates to `edge`.
```

Comment:
Can we standardise the terminology across files, and use "small" and "large" everywhere?

Reply:
Good idea! Fixed in 2c6bbd8060986a1611ccf497355d9d152416964b.